### ISSUE TITLE
googlesamples-assistant-hotword doesn't reply

### ISSUE THREAD

**Issue by wxgithub** [googlesamples/assistant-sdk-python#198]
I installed Google Assistant on an ARM board. pushtotalk sample app works fine, but if I run hotword sample, I couldn't get any reply. Do i need to change anything to make this to work? 

Console messages look normal:
(env) [root@home]# googlesamples-assistant-hotword --device_model_id my board
device_model_id: my board
device_id: E0185D24C6476E2498590375A1615FD7

[16353:16375:ERROR:assistant_nupnp_client.cc(103)] Cloud discovery nupnp request failed: [60] Peer certificate cannot be authenticated with given CA certificates
ON_MUTED_CHANGED:
  {u'is_muted': False}
ON_START_FINISHED
[16353:16375:ERROR:assistant_nupnp_client.cc(103)] Cloud discovery nupnp request failed: [60] Peer certificate cannot be authenticated with given CA certificates
[16353:16375:ERROR:assistant_nupnp_client.cc(103)] Cloud discovery nupnp request failed: [60] Peer certificate cannot be authenticated with given CA certificates

ON_CONVERSATION_TURN_STARTED
ON_END_OF_UTTERANCE
ON_RECOGNIZING_SPEECH_FINISHED:
  {u'text': u"what's the temperature in San Jose"}
ON_RESPONDING_STARTED:
  {u'is_error_response': False}
ON_RESPONDING_FINISHED
ON_CONVERSATION_TURN_FINISHED:
  {u'with_follow_on_turn': False}

**Comment by wxgithub** [comment]
No, not Pi. It is Samsung ARTIK530 board (https://www.artik.io/modules/artik-530/). Right, looks like everything works except for audio output. As mentioned, audio output works fine with pushtotalk sample. What are the differences between these 2 sample apps?

**Comment by Fleker** [comment]
In the hotword sample, the audio output is more tightly coupled to the underlying code, so it may have certain dependencies on ALSA output that are not an issue when using the more generic Service API.
